src/main/java/uk/gov/companieshouse/pscstatementdataapi/api/PscStatementApiService.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.companieshouse.pscstatementdataapi.api;
+
+import java.time.OffsetDateTime;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+import uk.gov.companieshouse.api.InternalApiClient;
+import uk.gov.companieshouse.api.chskafka.ChangedResource;
+import uk.gov.companieshouse.api.chskafka.ChangedResourceEvent;
+import uk.gov.companieshouse.api.error.ApiErrorResponseException;
+import uk.gov.companieshouse.api.handler.chskafka.request.PrivateChangedResourcePost;
+import uk.gov.companieshouse.api.model.ApiResponse;
+import uk.gov.companieshouse.api.psc.Statement;
+import uk.gov.companieshouse.logging.Logger;
+
+@Service
+public class PscStatementApiService {
+
+    @Autowired
+    ApiClientService apiClientService;
+    @Value("${psc-statements.api.resource.changed.uri}")
+    private String resourceChangedUri;
+    @Value("${psc-statements.api.resource.kind}")
+    private String resourceKind;
+    private static final String PSC_STATEMENTS_URI = "/company/%s/persons-with-significant-control-statements/%s/internal";
```

Comment:
I believe this uri will be used for a get request from the resource-changed-publisher to the data-api when event type is changed. Therefore should we omit the /internal now to be ahead?

Reply:
Yes, makes sense. I thought we'd only send a message to the resource-changed-publisher on Put and Delete, which both have "/internal" but I assume it will call Get through the PscStatementApiService on a Put